spec/acceptance_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'open3'
+
+describe 'Acceptance', integration: true do
+  it 'plays a draw game' do
+    stdout_str, _stderr_str, status = Open3.capture3('ruby bin/run', stdin_data: "1\n2\n3\n5\n4\n6\n8\n7\n9\n")
```

Comment:
**nitpick:** `Open3#capture2` would also work here since you're not (currently) writing anything to stderr 

That would also get rid of that ignored `stderr_str` variable.

Otherwise, nice job on finding this module and getting it hooked up for your test. Running something else in a subprocess isn't generally something you have to do all the time but it's good to have that as an option in your toolbox.

Reply:
Thank you! And I appreciate the tip about capture2 - the name makes sense now haha.